.terraform.lock.hcl

```diff
@@ -2,19 +2,20 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/hashicorp/aws" {
```

Comment:
not sure if it makes sense to have this file under version control at all for modules. we probbaly can ignore this in git